Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.4
+RUN apt-get update -qq && apt-get install -y nodejs postgresql-client
```

Comment:
Does this application need `nodejs` or `postgresql-client` to run?

If it doesn't, I guess these have been added as a convenience for development only. As this is still a Dockerfile that may end up being one day used in production, it seems to me that the safest option would be to install this package only when rarely needed:

```
docker-compose run --rm web bash
$ apt-get update -qq && apt-get install -y nodejs postgresql-client
$ … use tools to investigate
```